src/NzbDrone.Api.Test/Sonarr.Api.Test.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net6.0</TargetFrameworks>
+    <Platforms>AnyCPU;x86</Platforms>
```

Comment:
I'll drop the changes in the `.csproj`-files. Not sure how I managed to commit them.